Sign up form

Darius-collab opened the pull request:
Create three page facilitate the user to signup on website. 
The first page is page appear without any validation error,
The second page is shows the validation errrors,
The third page is shows if signup is succeeded.
For doing this I use html for structuring and css for styling it.

Darius-collab committed:
feat: User login form

- Create the form which helps the user to sign in.
- Use HTML for semmentic markup and CSS for styling.

Commit:
Testing commit

Commit:
Add test2

Commit:
remove testing file

Commit:
First login form

Commit:
First login form

Commit:
feat: Full login form

-Create the form which helps the user to sign in.
- Use HTML data structure and CSS for styling.

Commit:
feat: Full signup form. -Create three pages which help the user to signup. -Use HTML data structure and CSS for styling.

Commit:
feat: Admin dashboard
-Create dashboard which has two aside, one header and one section.
-Use picture file for make the users profile.
-Use HTML for structure and CSS for style.

descholar-ceo reviewed (changes requested):
Hi @Darius-collab 🤝 

Thank you for working on this project, however, your resulting project does not match with the original design you were provided in the original project specifications, you are advised to repeat the project in order to have a matching page design

| Design provided in project's specifications | Your Resulting design |
|----------|----------|
|<img width="652" alt="Screenshot 2024-04-29 at 1 31 57 PM" src="https://github.com/Darius-collab/online-examination/assets/56886064/3c031f00-aaf4-4b80-b33c-9881f9a6f8e2"> | <img width="1679" alt="Screenshot 2024-04-29 at 1 32 16 PM" src="https://github.com/Darius-collab/online-examination/assets/56886064/8acacd8b-d39e-405f-b3d1-a2352d9ebfd2">|

Commit:
feat: Rock Paper Scisssors game.
 -Create the rock paper scissors game.
 -Use the html for write their code and css for styling it.
 -Use the JavaScript code to make the logic which control the game.

Darius-collab committed:
Rock Paper Scissors

Commit:
Merge remote-tracking branch 'origin/Rock-Paper-Scissors' into ft-admin-dashboard
